tests/ert_tests/ert3/data/test_transmitters.py

```diff
@@ -173,7 +173,7 @@ async def test_simple_record_transmit_and_dump(
         transmitter = record_transmitter_factory(name="some_name")
         await transmitter.transmit_data(data_in)
 
-        await transmitter.dump("record.json")
```

Comment:
Maybe we should extend the test matrix here to include all the mime types as well, instead of passing it per test case?

Reply:
Good idea, will do.

Reply:
Solved this by introducing the `mime_types` parameter, but skipping all unsupported configurations. These are:

- binary data to non-binary format, e.g. `b'1'` to JSON.
- non-binary data, e.g. JSON to binary

There's nothing in ert3 currently needing this, so it isn't supported. And logically, it seems like it should stay this way.

What do you think?

Reply:
Perfect! :clap: Looks very good!